### ISSUE TITLE
Mac OS X binary releases?

### ISSUE THREAD

**Issue by wfaulk** [leonsoft-kras/transmisson-remote-gui#878]
You seemed to provide binary releases for Mac OS X through 5.0.1, but none since.  Is there any reason why you stopped?

**Comment by leonsoft-kras** [comment]
I am not the author of the project.
I have no mac os to make the next version of the distribution (of my improvements)

**Comment by nos1609** [comment]
Могу помочь со сборкой под макось. Есть xcode, последняя девелоперская макось и руки из плеч. Но компилил в основном под линух всё...

**Comment by leonsoft-kras** [comment]
Не возражаю. Если от меня что-то потребуется - напишите.
Все последние исходники - можно скачать без проблем )
Скрипты,файлы компиляции под Мак можно будет потом залить к исходникам.

**Comment by nos1609** [comment]
Поставил последние версии компиляторов, собираю по мануалу [отсюда](https://sourceforge.net/p/transgui/code/975/tree/wiki/Building.wiki) но жалуется на ошибки в самом коде. Lazarus 1.6, fpc и fpsrc 3.00. Косяк где-то у меня или проблема в свежем компиляторе?

`~/D/transmisson-remote-gui-5.2.0 ❯❯❯ make                                                                                                                                                                                                                                     ⏎
Using Lazarus dir: /Developer/lazarus/
/bin/mkdir -p units/i386-darwin
/Developer/lazarus//tools/lazres about.lrs about.lfm
about.lfm ResourceName='TAboutForm' Type='FORMDATA'
/Developer/lazarus//tools/lazres addlink.lrs addlink.lfm
addlink.lfm ResourceName='TAddLinkForm' Type='FORMDATA'
/Developer/lazarus//tools/lazres addtorrent.lrs addtorrent.lfm
addtorrent.lfm ResourceName='TAddTorrentForm' Type='FORMDATA'
/Developer/lazarus//tools/lazres addtracker.lrs addtracker.lfm
addtracker.lfm ResourceName='TAddTrackerForm' Type='FORMDATA'
/Developer/lazarus//tools/lazres baseform.lrs baseform.lfm
baseform.lfm ResourceName='TBaseForm' Type='FORMDATA'
/Developer/lazarus//tools/lazres colsetup.lrs colsetup.lfm
colsetup.lfm ResourceName='TColSetupForm' Type='FORMDATA'
/Developer/lazarus//tools/lazres connoptions.lrs connoptions.lfm
connoptions.lfm ResourceName='TConnOptionsForm' Type='FORMDATA'
/Developer/lazarus//tools/lazres daemonoptions.lrs daemonoptions.lfm
daemonoptions.lfm ResourceName='TDaemonOptionsForm' Type='FORMDATA'
/Developer/lazarus//tools/lazres download.lrs download.lfm
download.lfm ResourceName='TDownloadForm' Type='FORMDATA'
/Developer/lazarus//tools/lazres main.lrs main.lfm
main.lfm ResourceName='TMainForm' Type='FORMDATA'
/Developer/lazarus//tools/lazres movetorrent.lrs movetorrent.lfm
movetorrent.lfm ResourceName='TMoveTorrentForm' Type='FORMDATA'
/Developer/lazarus//tools/lazres options.lrs options.lfm
options.lfm ResourceName='TOptionsForm' Type='FORMDATA'
/Developer/lazarus//tools/lazres torrprops.lrs torrprops.lfm
torrprops.lfm ResourceName='TTorrPropsForm' Type='FORMDATA'
/usr/local/bin/ppc386 -MObjFPC -dLCL -dLCLcarbon -O2 -g- -CX -XX -Xs  -k-macosx_version_min -k10.5 -XR/Developer/SDKs/MacOSX10.5.sdk/ -Fusynapse/source/lib -Fujson -Fu/Developer/lazarus//lcl/units/i386-darwin -Fu/Developer/lazarus//lcl/units/i386-darwin/carbon -Fu/Developer/lazarus//components/lazutils/lib/i386-darwin -Fu/usr/local/lib/fpc/3.0.0/units/i386-darwin/rtl -FE. -FUunits/i386-darwin -di386 transgui.lpr
Free Pascal Compiler version 3.0.0 [2015/11/14] for i386
Copyright (c) 1993-2015 by Florian Klaempfl and others
Target OS: Darwin for i386
Compiling transgui.lpr
Compiling maclocale.pas
Assembling (pipe) units/i386-darwin/maclocale.s
Compiling baseform.pas
Compiling vargrid.pas
Compiling varlist.pas
Assembling (pipe) units/i386-darwin/varlist.s
vargrid.pas(543,7) Warning: Symbol "UTF8UpperCase" is deprecated: "Use the function in LazUTF8 unit"
vargrid.pas(549,10) Warning: Symbol "UTF8UpperCase" is deprecated: "Use the function in LazUTF8 unit"
vargrid.pas(549,39) Warning: Implicit string type conversion with potential data loss from "WideString" to "AnsiString"
vargrid.pas(1082,27) Warning: Implicit string type conversion with potential data loss from "WideString" to "AnsiString"
vargrid.pas(1414,15) Warning: Implicit string type conversion with potential data loss from "WideString" to "AnsiString"
Assembling (pipe) units/i386-darwin/vargrid.s
Assembling (pipe) units/i386-darwin/baseform.s
Compiling main.pas
Compiling ./synapse/source/lib/httpsend.pas
Compiling ./synapse/source/lib/blcksock.pas
Compiling ./synapse/source/lib/synafpc.pas
Assembling (pipe) units/i386-darwin/synafpc.s
Compiling ./synapse/source/lib/synsock.pas
ssfpc.inc(714,26) Warning: Symbol "HostToNet" is deprecated
ssfpc.inc(824,18) Warning: Symbol "HostToNet" is deprecated
ssfpc.inc(884,25) Warning: Symbol "NetToHost" is deprecated
Assembling (pipe) units/i386-darwin/synsock.s
Compiling ./synapse/source/lib/synautil.pas
synautil.pas(637,96) Warning: Symbol "TimeSeparator" is deprecated
synautil.pas(1981,3) Note: Local variable "BackStop" is assigned but never used
synautil.pas(2121,88) Warning: Symbol "ShortMonthNames" is deprecated
synautil.pas(2122,87) Warning: Symbol "ShortMonthNames" is deprecated
Assembling (pipe) units/i386-darwin/synautil.s
Compiling ./synapse/source/lib/synacode.pas
Assembling (pipe) units/i386-darwin/synacode.s
Compiling ./synapse/source/lib/synaip.pas
Assembling (pipe) units/i386-darwin/synaip.s
Assembling (pipe) units/i386-darwin/blcksock.s
Assembling (pipe) units/i386-darwin/httpsend.s
Compiling rpc.pas
Compiling ./synapse/source/lib/ssl_openssl.pas
Compiling ./synapse/source/lib/ssl_openssl_lib.pas
Assembling (pipe) units/i386-darwin/ssl_openssl_lib.s
Assembling (pipe) units/i386-darwin/ssl_openssl.s
Compiling main.pas
Compiling ipresolver.pas
Compiling GeoIP.pas
Compiling utils.pas
utils.pas(242,11) Warning: Symbol "ParamStrUTF8" is deprecated: "Use the function in LazUTF8 unit"
utils.pas(308,6) Warning: Symbol "FileExistsUTF8" is deprecated: "Use the function in LazFileUtils unit"
utils.pas(328,6) Warning: Symbol "FileExistsUTF8" is deprecated: "Use the function in LazFileUtils unit"
utils.pas(333,74) Warning: Symbol "SysErrorMessageUTF8" is deprecated: "Use the function in LazUTF8 unit"
utils.pas(428,16) Warning: Symbol "CommandLine" is deprecated
utils.pas(505,56) Warning: Implicit string type conversion with potential data loss from "UnicodeString" to "AnsiString"
utils.pas(505,40) Warning: Implicit string type conversion with potential data loss from "UnicodeString" to "AnsiString"
Assembling (pipe) units/i386-darwin/utils.s
Assembling (pipe) units/i386-darwin/GeoIP.s
Assembling (pipe) units/i386-darwin/ipresolver.s
Compiling restranslator.pas
restranslator.pas(156,6) Warning: Symbol "FindFirstUTF8" is deprecated: "Use the function in LazFileUtils unit"
restranslator.pas(169,13) Warning: Symbol "FindNextUTF8" is deprecated: "Use the function in LazFileUtils unit"
restranslator.pas(183,33) Warning: Symbol "ExtractFileNameOnly" is deprecated: "Use the function in LazFileUtils unit"
restranslator.pas(183,13) Warning: Symbol "FindFirstUTF8" is deprecated: "Use the function in LazFileUtils unit"
restranslator.pas(195,26) Warning: Symbol "ExtractFileNameOnly" is deprecated: "Use the function in LazFileUtils unit"
restranslator.pas(199,34) Warning: Symbol "ExtractFileNameOnly" is deprecated: "Use the function in LazFileUtils unit"
restranslator.pas(246,3) Warning: Symbol "LCLGetLanguageIDs" is deprecated: "Use function LazGetLanguageIDs in LazUTF8 unit"
restranslator.pas(248,6) Warning: Symbol "ExtractFileNameOnly" is deprecated: "Use the function in LazFileUtils unit"
restranslator.pas(249,28) Warning: Symbol "FileExistsUTF8" is deprecated: "Use the function in LazFileUtils unit"
restranslator.pas(302,61) Warning: Symbol "ExtractFileNameOnly" is deprecated: "Use the function in LazFileUtils unit"
restranslator.pas(303,8) Warning: Symbol "FileExistsUTF8" is deprecated: "Use the function in LazFileUtils unit"
restranslator.pas(316,11) Warning: Symbol "FileExistsUTF8" is deprecated: "Use the function in LazFileUtils unit"
restranslator.pas(338,3) Warning: Symbol "LCLGetLanguageIDs" is deprecated: "Use function LazGetLanguageIDs in LazUTF8 unit"
restranslator.pas(350,59) Warning: Symbol "ExtractFileNameOnly" is deprecated: "Use the function in LazFileUtils unit"
restranslator.pas(385,6) Warning: Symbol "FileExistsUTF8" is deprecated: "Use the function in LazFileUtils unit"
restranslator.pas(493,28) Warning: Symbol "UTF8UpperCase" is deprecated: "Use the function in LazUTF8 unit"
restranslator.pas(493,46) Warning: Symbol "UTF8UpperCase" is deprecated: "Use the function in LazUTF8 unit"
restranslator.pas(656,8) Warning: Symbol "FileExistsUTF8" is deprecated: "Use the function in LazFileUtils unit"
Assembling (pipe) units/i386-darwin/restranslator.s
Compiling addtorrent.pas
Compiling main.pas
Compiling urllistenerosx.pas
Assembling (pipe) units/i386-darwin/urllistenerosx.s
Compiling connoptions.pas
Writing Resource String Table file: connoptions.rsj
Assembling (pipe) units/i386-darwin/connoptions.s
Compiling torrprops.pas
Assembling (pipe) units/i386-darwin/torrprops.s
Compiling daemonoptions.pas
daemonoptions.pas(268,17) Warning: Symbol "SysToUTF8" is deprecated: "Use the function in LazUTF8 unit"
Writing Resource String Table file: daemonoptions.rsj
Assembling (pipe) units/i386-darwin/daemonoptions.s
Compiling about.pas
Writing Resource String Table file: about.rsj
Assembling (pipe) units/i386-darwin/about.s
Compiling download.pas
Writing Resource String Table file: download.rsj
Assembling (pipe) units/i386-darwin/download.s
Compiling colsetup.pas
Assembling (pipe) units/i386-darwin/colsetup.s
Compiling addlink.pas
Writing Resource String Table file: addlink.rsj
Assembling (pipe) units/i386-darwin/addlink.s
Compiling movetorrent.pas
Writing Resource String Table file: movetorrent.rsj
Assembling (pipe) units/i386-darwin/movetorrent.s
Compiling addtracker.pas
Writing Resource String Table file: addtracker.rsj
Assembling (pipe) units/i386-darwin/addtracker.s
Compiling options.pas
Assembling (pipe) units/i386-darwin/options.s
Compiling bencode.pas
Assembling (pipe) units/i386-darwin/bencode.s
main.pas(923,8) Warning: Symbol "FileExistsUTF8" is deprecated: "Use the function in LazFileUtils unit"
main.pas(983,48) Warning: Symbol "FileExistsUTF8" is deprecated: "Use the function in LazFileUtils unit"
main.pas(987,8) Warning: Symbol "FileExistsUTF8" is deprecated: "Use the function in LazFileUtils unit"
main.pas(1062,8) Warning: Symbol "FileExistsUTF8" is deprecated: "Use the function in LazFileUtils unit"
main.pas(1065,46) Warning: Symbol "GetAppConfigDirUTF8" is deprecated: "Use the function in LazFileUtils unit"
main.pas(1069,3) Warning: Symbol "ForceDirectoriesUTF8" is deprecated: "Use the function in LazFileUtils unit"
main.pas(1077,6) Warning: Symbol "FileExistsUTF8" is deprecated: "Use the function in LazFileUtils unit"
main.pas(1079,10) Warning: Comparison might be always true due to range of constant and expression
main.pas(1083,9) Warning: Symbol "DeleteFileUTF8" is deprecated: "Use the function in LazFileUtils unit"
main.pas(1089,34) Warning: Symbol "FileExistsUTF8" is deprecated: "Use the function in LazFileUtils unit"
main.pas(1093,6) Warning: Symbol "FileExistsUTF8" is deprecated: "Use the function in LazFileUtils unit"
main.pas(1103,12) Warning: Symbol "FileExistsUTF8" is deprecated: "Use the function in LazFileUtils unit"
main.pas(1106,14) Warning: Symbol "FileExistsUTF8" is deprecated: "Use the function in LazFileUtils unit"
main.pas(1114,5) Warning: Symbol "DeleteFileUTF8" is deprecated: "Use the function in LazFileUtils unit"
main.pas(1120,7) Warning: Symbol "DeleteFileUTF8" is deprecated: "Use the function in LazFileUtils unit"
main.pas(1329,6) Warning: Symbol "FileExistsUTF8" is deprecated: "Use the function in LazFileUtils unit"
main.pas(1530,3) Warning: Symbol "DeleteFileUTF8" is deprecated: "Use the function in LazFileUtils unit"
main.pas(1714,67) Warning: Implicit string type conversion with potential data loss from "WideString" to "TTranslateString"
main.pas(1719,10) Warning: Implicit string type conversion with potential data loss from "WideString" to "AnsiString"
main.pas(1760,101) Warning: Implicit string type conversion with potential data loss from "WideString" to "AnsiString"
main.pas(2012,16) Warning: Symbol "SysToUTF8" is deprecated: "Use the function in LazUTF8 unit"
main.pas(2052,11) Warning: Symbol "DeleteFileUTF8" is deprecated: "Use the function in LazFileUtils unit"
main.pas(2436,13) Warning: Symbol "DeleteFileUTF8" is deprecated: "Use the function in LazFileUtils unit"
main.pas(2515,3) Warning: Symbol "ForceDirectoriesUTF8" is deprecated: "Use the function in LazFileUtils unit"
main.pas(2531,12) Warning: Symbol "FileExistsUTF8" is deprecated: "Use the function in LazFileUtils unit"
main.pas(2551,9) Warning: Symbol "DeleteFileUTF8" is deprecated: "Use the function in LazFileUtils unit"
main.pas(2556,14) Warning: Symbol "FileExistsUTF8" is deprecated: "Use the function in LazFileUtils unit"
main.pas(2640,8) Warning: Symbol "SysToUTF8" is deprecated: "Use the function in LazUTF8 unit"
main.pas(2641,10) Warning: Symbol "FileExistsUTF8" is deprecated: "Use the function in LazFileUtils unit"
main.pas(2663,5) Warning: Symbol "DeleteFileUTF8" is deprecated: "Use the function in LazFileUtils unit"
main.pas(2665,5) Warning: Symbol "DeleteFileUTF8" is deprecated: "Use the function in LazFileUtils unit"
main.pas(2666,5) Warning: Symbol "DeleteFileUTF8" is deprecated: "Use the function in LazFileUtils unit"
main.pas(3665,3) Warning: Symbol "DeleteFileUTF8" is deprecated: "Use the function in LazFileUtils unit"
main.pas(3853,12) Warning: Implicit string type conversion from "AnsiString" to "UnicodeString"
main.pas(3853,73) Warning: Implicit string type conversion with potential data loss from "UnicodeString" to "AnsiString"
main.pas(3858,10) Warning: Symbol "FileExistsUTF8" is deprecated: "Use the function in LazFileUtils unit"
main.pas(3858,31) Warning: Symbol "DirectoryExistsUTF8" is deprecated: "Use the function in LazFileUtils unit"
main.pas(3896,6) Warning: Symbol "UTF8UpperCase" is deprecated: "Use the function in LazUTF8 unit"
main.pas(3901,20) Warning: Symbol "UTF8UpperCase" is deprecated: "Use the function in LazUTF8 unit"
main.pas(3901,49) Warning: Implicit string type conversion with potential data loss from "WideString" to "AnsiString"
main.pas(3918,101) Warning: Implicit string type conversion with potential data loss from "WideString" to "AnsiString"
main.pas(3979,42) Warning: Implicit string type conversion from "AnsiString" to "UnicodeString"
main.pas(4573,22) Warning: Symbol "DirectoryExistsUTF8" is deprecated: "Use the function in LazFileUtils unit"
main.pas(4576,10) Warning: Symbol "DirectoryExistsUTF8" is deprecated: "Use the function in LazFileUtils unit"
main.pas(4831,10) Warning: Implicit string type conversion from "AnsiString" to "WideString"
main.pas(4839,16) Warning: Implicit string type conversion from "AnsiString" to "WideString"
main.pas(4849,35) Warning: Implicit string type conversion from "AnsiString" to "UnicodeString"
main.pas(4849,42) Warning: Implicit string type conversion with potential data loss from "UnicodeString" to "AnsiString"
main.pas(4876,13) Warning: Implicit string type conversion with potential data loss from "WideString" to "AnsiString"
main.pas(5068,19) Warning: Implicit string type conversion with potential data loss from "WideString" to "AnsiString"
main.pas(5072,22) Warning: Implicit string type conversion with potential data loss from "WideString" to "AnsiString"
main.pas(5098,61) Warning: Implicit string type conversion with potential data loss from "WideString" to "AnsiString"
main.pas(5274,10) Warning: Implicit string type conversion with potential data loss from "WideString" to "AnsiString"
main.pas(5338,12) Warning: Implicit string type conversion with potential data loss from "WideString" to "AnsiString"
main.pas(5373,20) Warning: Symbol "UTF8UpperCase" is deprecated: "Use the function in LazUTF8 unit"
main.pas(5373,50) Warning: Symbol "UTF8UpperCase" is deprecated: "Use the function in LazUTF8 unit"
main.pas(5373,99) Warning: Implicit string type conversion with potential data loss from "WideString" to "AnsiString"
main.pas(5373,12) Warning: Symbol "UTF8Pos" is deprecated: "Use the function in LazUTF8 unit"
main.pas(5430,71) Warning: Implicit string type conversion from "AnsiString" to "UnicodeString"
main.pas(5703,27) Warning: Implicit string type conversion with potential data loss from "WideString" to "AnsiString"
main.pas(5842,68) Warning: Implicit string type conversion with potential data loss from "WideString" to "TTranslateString"
main.pas(5869,6) Warning: Implicit string type conversion with potential data loss from "WideString" to "AnsiString"
main.pas(6419,10) Warning: Symbol "FileExistsUTF8" is deprecated: "Use the function in LazFileUtils unit"
main.pas(6448,6) Warning: Symbol "FileExistsUTF8" is deprecated: "Use the function in LazFileUtils unit"
main.pas(6448,34) Warning: Symbol "DirectoryExistsUTF8" is deprecated: "Use the function in LazFileUtils unit"
main.pas(6640,90) Warning: Implicit string type conversion with potential data loss from "WideString" to "TTranslateString"
main.pas(6860,5) Warning: Symbol "DeleteFileUTF8" is deprecated: "Use the function in LazFileUtils unit"
main.pas(6915,24) Warning: Symbol "UTF8LowerCase" is deprecated: "Use the function in LazUTF8 unit"
main.pas(6915,53) Warning: Symbol "UTF8LowerCase" is deprecated: "Use the function in LazUTF8 unit"
Writing Resource String Table file: main.rsj
Assembling (pipe) units/i386-darwin/main.s
addtorrent.pas(235,10) Warning: Implicit string type conversion with potential data loss from "UnicodeString" to "AnsiString"
addtorrent.pas(438,13) Warning: Implicit string type conversion from "AnsiString" to "UnicodeString"
addtorrent.pas(438,20) Warning: Implicit string type conversion with potential data loss from "UnicodeString" to "AnsiString"
addtorrent.pas(445,15) Warning: Implicit string type conversion from "AnsiString" to "UnicodeString"
addtorrent.pas(445,22) Warning: Implicit string type conversion with potential data loss from "UnicodeString" to "AnsiString"
addtorrent.pas(777,105) Warning: Implicit string type conversion from "AnsiString" to "UnicodeString"
addtorrent.pas(777,103) Warning: Implicit string type conversion with potential data loss from "UnicodeString" to "AnsiString"
addtorrent.pas(824,6) Warning: Symbol "UTF8UpperCase" is deprecated: "Use the function in LazUTF8 unit"
addtorrent.pas(830,20) Warning: Symbol "UTF8UpperCase" is deprecated: "Use the function in LazUTF8 unit"
addtorrent.pas(830,49) Warning: Implicit string type conversion with potential data loss from "WideString" to "AnsiString"
Writing Resource String Table file: addtorrent.rsj
Assembling (pipe) units/i386-darwin/addtorrent.s
rpc.pas(266,11) Warning: Implicit string type conversion from "AnsiString" to "WideString"
Writing Resource String Table file: rpc.rsj
Assembling (pipe) units/i386-darwin/rpc.s
Assembling (pipe) units/i386-darwin/transgui.s
transgui.lpr(50,1) Error: Can't open resource file "/Users/nos1609/Downloads/transmisson-remote-gui-5.2.0/transgui.res"
transgui.lpr(50,1) Fatal: There were 1 errors compiling module, stopping
Fatal: Compilation aborted
make: *** [transgui] Error 1`

**Comment by leonsoft-kras** [comment]
ошибка тут только одна - на transgui.res. 
в винде этот файл создается при компиляции. соотв-но он должен у вас создаться в процессе компиляции. В чем причина его отсутствия - не подскажу. Мб посмотреть свойства проекта "под МАКом"?

предупреждения не влияют на компиляцию и создание исполнимого кода. надо кстати посмотреть, какие предупреждения при компиляции под виндовс.

**Comment by leonsoft-kras** [comment]
попробуйте посмотреть, мб у вас есть данный res файл на диске? если да - удалить его и выполнить полное построение? + глянуть настройки проекта.

**Comment by nos1609** [comment]
![2016-03-01 17 04 55](https://cloud.githubusercontent.com/assets/11856987/13429150/ce68fe4c-dfcf-11e5-801e-f671637f3339.png)
Это если через гуи собирать, в самом Lazarus

**Comment by leonsoft-kras** [comment]
такая же хрень есть под вендой.
я нашел способ: "Запуск" - "Очистить и собрать". 
выполняется перекомпиляция всего на свете и это сообщение больше не возникает. после этого всё компилируется нормально

**Comment by leonsoft-kras** [comment]
такая ситуация была в старом фрипаскале и в последних версиях.

з.ы. вообще паскалем я занимался лет 20-25 назад более-менее плотно, потом был фортран. а последние 18 лет - С++, С#. так что мне несколько сложно разбираться в некоторых наворотах фрипаскаля и данном проекте. В общем чем смогу - помогу.

**Comment by Proper-Job** [comment]
It'd be great to have this discussion in English.  I tried building on Mac OS 10.10 using Lazarus 1.6 and fpc and fpcsrc 3.0.0 and I'm getting the same error message.
Is there a sulution to this?

**Comment by leonsoft-kras** [comment]
TVarGrid error similarly occurs when compiling on Windows.
I chose the menu "Run - Clean up and build..." (in Lazarus). All settings left at default.
Try. I did not do any special corrections in the project (I am not the author)

Please write the result

**Comment by leonsoft-kras** [comment]
nos1609! 
there results?

**Comment by nlunardelli** [comment]
Hi, install Lazarus 1.6, fpcsrc-3.0.0 and fps-3.0.0, how should I proceed after "Run - Clean up and build..."?
I'm a newbie, rsrsrs

**Comment by leonsoft-kras** [comment]
![11111](https://cloud.githubusercontent.com/assets/11468577/14127378/788a8b84-f641-11e5-88d5-3af37dfbe0bc.png)
![222222](https://cloud.githubusercontent.com/assets/11468577/14127382/7a977dec-f641-11e5-922c-1c5f275c911e.png)

**Comment by leonsoft-kras** [comment]
I use Lazarus 1.4 but I think the action in version 1.6 are the same

**Comment by nlunardelli** [comment]
Here are the sequence os Screem Shot's
Looks like my problem is with Debugger in Lazarus, any clue how to fix it?

![screen shot 1](https://cloud.githubusercontent.com/assets/191792/14176057/6f6fb134-f725-11e5-9589-f76db89b1f97.png)
![screen shot 3](https://cloud.githubusercontent.com/assets/191792/14176062/6f80ada4-f725-11e5-8704-414e0d41750e.png)
![screen shot 4](https://cloud.githubusercontent.com/assets/191792/14176060/6f7bb178-f725-11e5-8d9d-d6870aabe669.png)
![screen shot 5](https://cloud.githubusercontent.com/assets/191792/14176059/6f75e54a-f725-11e5-8669-87815bff422c.png)
![screen shot 6](https://cloud.githubusercontent.com/assets/191792/14176061/6f7bbb3c-f725-11e5-8bf4-d3406c34325c.png)

**Comment by nos1609** [comment]
@nlunardelli you have to [download and install gdb](https://stackoverflow.com/questions/33162757/how-to-install-gdb-debugger-in-mac-osx-el-capitan)

**Comment by Snegohod** [comment]
[Application version 5.3.0](https://github.com/leonsoft-kras/transmisson-remote-gui/issues/895#issuecomment-204820087)

**Comment by nos1609** [comment]
Работает.
It works.

**Comment by leonsoft-kras** [comment]
thanks!

**Comment by Campacavallo** [comment]
It works perfectly (updated from the old 5.0.1 version). Thanks!

**Comment by nlunardelli** [comment]
@Snegohod Thanks!
After downloading I tried to open, but the app closes rapidly.

I tried to run /Users/Nico/Downloads/Transmission Remote GUI.app/Contents/MacOS/transgui
and got the following result

```
Last login: Thu Mar 31 18:46:35 on ttys000
iMac:~ XXXX$ /Users/$$$$/Downloads/Transmission\ Remote\ GUI.app/Contents/MacOS/transgui ; exit;
[FORMS.PP] ExceptionOccurred 
  Sender=EFCreateError
  Exception=Unable to create file "/Users/Nico/.config/Transmission Remote GUI/transgui.ini"
  Stack trace:
  $0007C644
  $0007CABC
  $0004E77A
TApplication.HandleException Unable to create file "/Users/Nico/.config/Transmission Remote GUI/transgui.ini"
  Stack trace:
  $0007C644
  $0007CABC
  $0004E77A
Exception at 0007C644: EFCreateError:
Unable to create file "/Users/Nico/.config/Transmission Remote GUI/transgui.ini".
logout
Saving session...
...copying shared history...
...saving history...truncating history files...
...completed.
Deleting expired sessions...none found.

[Process completed]
```

How can I fix it?

Thanks

**Comment by Snegohod** [comment]
@nlunardelli help says

```
Description

EFCreateError is thrown when an application unsuccessfully attempts to create a file. This can occur, for example, if a user specifies an invalid file name, or specifies the name of an existing file that cannot be overwritten because the user lacks appropriate access permission.
```
File /Users/Nico/.config/Transmission Remote GUI/transgui.ini exists?
If yes, try to check permissions of the file or backup it and delete.
Then start application.

**Comment by nlunardelli** [comment]
`File /Users/Nico/.config/Transmission Remote GUI/transgui.ini exists?`
No, does not exist.

Any clue how to fix it?
Thanks

**Comment by Snegohod** [comment]
Try to start as root
Open terminal and execute
> sudo /Users/Nico/Downloads/Transmission\ Remote\ GUI.app/Contents/MacOS/transgui

**Comment by nlunardelli** [comment]
@Snegohod 

here is the terminal:
```
Nicos-iMac21-4:~ Nico$ sudo /Users/Nico/Downloads/Transmission\ Remote\ GUI.app/Contents/MacOS/transgui
Password:
Warning: TWinControl.DestroyHandle panReconnect:TPanel Handle not Allocated
Warning: TWinControl.DestroyHandle panDetailsWait:TPanel Handle not Allocated
```

It start running, when I tried to paste a magnet link it closed and stopped opening

**Comment by Snegohod** [comment]
@nlunardelli could you please give me more information about the installed OS, folder permissions and etc.
I checked on a clean installed OS X 10.11 and it`s works perfectly.
Try to inslall old version 5.0.1 from [here](https://sourceforge.net/projects/transgui/files/5.0.1/)

**Comment by nlunardelli** [comment]
@Snegohod Thanks in advanced for your patient  and help.

I have downloaded the 5.0.1 version and after the install it makes the same error, closes after it opens.
Than I tried to open it with de SUDO command as you mentioned above, and it opened, but it is working oddly.....

![screen shot 2016-04-07 at 11 40 18 am](https://cloud.githubusercontent.com/assets/191792/14355349/00328b60-fcb7-11e5-8bfe-5238a8dd1342.png)
![screen shot 2016-04-07 at 11 22 40 am](https://cloud.githubusercontent.com/assets/191792/14355364/0a67413e-fcb7-11e5-9331-a7d3846e3065.png)
![screen shot 2016-04-07 at 11 19 44 am](https://cloud.githubusercontent.com/assets/191792/14355379/11d82d2a-fcb7-11e5-86b3-a9eb63c8ff9f.png)
![screen shot 2016-04-07 at 11 19 35 am](https://cloud.githubusercontent.com/assets/191792/14355382/15e721a0-fcb7-11e5-8b8a-a385c29a3c1e.png)
![screen shot 2016-04-07 at 11 11 43 am](https://cloud.githubusercontent.com/assets/191792/14355507/938473c4-fcb7-11e5-829a-89f8a88ad6ca.png)

**Comment by Snegohod** [comment]
I looked permissions at my test OS X 10.11.4 and compare it with your screenshots
Try to change permissions group staff to Read & Write
All else is the same

**Comment by nlunardelli** [comment]
@Snegohod Thanks again al versions working now!!!!

After changing staff on the transgui.ini stared to work.
![screen shot 2016-04-07 at 4 11 34 pm](https://cloud.githubusercontent.com/assets/191792/14363665/c9a919cc-fcdb-11e5-8968-963a5c070ef2.png)

**Comment by Snegohod** [comment]
@nlunardelli You are welcome

**Comment by y** [comment]
There is no mac binary for the latest release, 5.40.

**Comment by Snegohod** [comment]
@y I know it
I compile new version after the holidays when I get home

**Comment by Snegohod** [comment]
Add app to release 5.4.0

**Comment by Snegohod** [comment]
@leonsoft-kras after commit https://github.com/leonsoft-kras/transmisson-remote-gui/commit/9f7ef0183eccfc392c806de9835ee1e38b79ff64 can not compile without errors on OS X, when I returned `<GenerateDebugInfo Value="False"/>` everything became as before. 
I think it's because gdb has been replaced by lldb, and is no longer supported, but lazarus use only gdb.
I know that I can install gdb but I do not need it. I do not use a debugger.
